conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -43,9 +43,9 @@ install:
     - cmd: conda config --add channels {{ channel }}{% endfor %}
 
     # Configure the VM.
-    - cmd: conda install -n root --quiet --yes conda-forge-build-setup
+    - cmd: appveyor-retry conda install -n root --quiet --yes conda-forge-build-setup
     {% if build_setup -%}
-    {{ build_setup }}{% endif %}
+    appveyor-retry {{ build_setup }}{% endif %}
```

Comment:
I think this has to be fixed in the python code.

Reply:
Do you mean here:
https://github.com/conda-forge/conda-forge-build-setup-feedstock/blob/master/recipe/run_conda_forge_build_setup_win.bat
?

In general, errors in build_setup cannot be due to the recipe, can they?

Reply:
Nope, [here](https://github.com/conda-forge/conda-smithy/blob/master/conda_smithy/configure_feedstock.py#L497) and [here](https://github.com/conda-forge/conda-smithy/blob/master/conda_smithy/configure_feedstock.py#L502)

Reply:
https://github.com/conda-forge/conda-smithy/blob/master/conda_smithy/configure_feedstock.py#L502 is not Windows-specific, no?

Reply:
It is. That code is inside the function `render_appveyor`.